Themis/Views/Assessment/CorrectionSidebar/FeedbackCellView.swift

```diff
@@ -70,7 +72,8 @@ struct FeedbackCellView: View {
                 cvm: cvm,
                 type: feedback.type,
                 showSheet: $showEditFeedback,
-                idForUpdate: feedback.id
+                idForUpdate: feedback.id,
+                exercise: nil
```

Comment:
Because of this, I have no choice of guidelines when editing a feedback. Is this intended? If yes, we might completely leave out the property on the `EditFeedbackSheet`.

<img width="809" alt="screenshot-2023-01-22_000934" src="https://user-images.githubusercontent.com/9006596/213944535-ca52e6a2-1913-4a4b-8ec6-a26ee522fe36.png">

Reply:
Not intended, just a leftover from testing. Should be resolved now